docs/install.md

```diff
@@ -29,13 +29,13 @@ each step in more detail.
 
 When setting up a cluster, the administrator must install the PMEM
 hardware on nodes and configure some or all of those instances of PMEM for usage by
-PMEM-CSI ([prerequisites](#prerequisites)]. Nodes where PMEM-CSI is
+PMEM-CSI ([prerequisites](#prerequisites)). Nodes where PMEM-CSI is
 supposed to run must have a certain [label in
 Kubernetes](#installation-and-setup).
 
 The administrator must install PMEM-CSI, using [the PMEM-CSI
 operator](#install-using-the-operator) (recommended) or with [scripts
-and YAML files in the source code](#install-from-source). The default
+and YAML files in the source code](#install-via-yaml-files). The default
```

Comment:
We really need some automatic checking for this :cry: 

Thanks for all the fixes.